AlfrescoApp/Model/Managers/RealmSyncManager+Refresh.m

```diff
@@ -184,7 +183,8 @@ - (void)cleanDataBaseOfUnwantedNodes
             [self.fileManager removeItemAtPath:filePath error:&deleteError];
             
             // Remove sync status.
-            [self removeSyncNodeStatusForNodeWithId:node.syncNodeInfoId inSyncNodesStatus:self.syncNodesStatus];
+            SyncOperationQueueManager *syncOpQM = self.syncQueues[[AccountManager sharedManager].selectedAccount.accountIdentifier];
```

Comment:
It would be nice to have a short named method that returns the SyncOperationQueueManager for the selected account.